cmake/3rdParty/Platform/Android/BuiltInPackages_android.cmake

```diff
@@ -30,10 +30,3 @@ ly_associate_package(PACKAGE_NAME OpenSSL-1.1.1o-rev2-android
 ly_associate_package(PACKAGE_NAME zlib-1.2.11-rev5-android                          TARGETS ZLIB                    PACKAGE_HASH 73c9e88892c237a3fc6eafc04268ccd9d479e6d55f9df2ed58b236c8f9cf2cae)
 ly_associate_package(PACKAGE_NAME lz4-1.9.4-rev1-android                            TARGETS lz4                     PACKAGE_HASH 97a4758f07bea6792dc68d7e9b84952800628bc482b33d9a93e2c52f9e758662)
 ly_associate_package(PACKAGE_NAME vulkan-validationlayers-1.3.261-rev1-android      TARGETS vulkan-validationlayers PACKAGE_HASH bb0742fef8e46069e027b57534da71c6cd176d9b3e129f4cf2a2194e3fad75ea)
-
-set(AZ_USE_PHYSX5 OFF CACHE BOOL "When ON PhysX Gem will use PhysX 5 SDK")
-if(AZ_USE_PHYSX5)
-    ly_associate_package(PACKAGE_NAME PhysX-5.1.1-rev1-android           TARGETS PhysX       PACKAGE_HASH 38b469c334017c3dd0a75ddd98c562846bacd017023a37aa26d56b747fb1440b)
-else()
-    ly_associate_package(PACKAGE_NAME PhysX-4.1.2.29882248-rev5-android  TARGETS PhysX       PACKAGE_HASH b346e8f9bc55f367a97d781d94c8a5c3bff8059478b8a7007e5fd17708dc1d07)
-endif()
```

Comment:
@spham-amzn This code is still present in iOS platform. Maybe it should be removed as well from the iOS file (https://github.com/o3de/o3de/blob/development/cmake/3rdParty/Platform/iOS/BuiltInPackages_ios.cmake#L33). All the other platforms have been removed except for the iOS one.

Reply:
Nice catch Aaron!